### PR TITLE
Updated Russian translations

### PR DESCRIPTION
**Pull request by ZXLTRXN**
Added some prases for ru locale which is necessary for using in mobile sdk.

### CODE REVIEW

File: android/sdk/src/main/res/values-ru/strings.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+<resources>
+    <string name="app_name">Jitsi Meet SDK</string>
+    <string name="dropbox_app_key"></string>
+    <string name="ongoing_notification_title">Текущая встреча</string>
+    <string name="ongoing_notification_text">Нажмите, чтобы вернуться к встрече.</string>
+    <string name="ongoing_notification_action_hang_up">Отключиться</string>
+    <string name="ongoing_notification_action_mute">Отключить звук</string>
+    <string name="ongoing_notification_action_unmute">Включить звук</string>
+    <string name="ongoing_notification_channel_name">Ongoing Conference Notifications</string>
+</resources>
```
Comment (damencho) [inline android/sdk/src/main/res/values-ru/strings.xml:11]: @saghul do we add translations to the android resource values?

**Comment by damencho** [comment]
Drop all empty translations, they would at least default to english.

File: android/sdk/src/main/res/values-ru/strings.xml (hunk 1 above)
Comment (saghul) [reply android/sdk/src/main/res/values-ru/strings.xml:11]: No, but it doesn't hurt either.

**Comment by saghul** [comment]
A couple of comments, LGTM otherwise.

**Review by saghul — APPROVED** [review]

**Comment by saghul** [comment]
Cheers!

**Pull request merged**